tox.ini

```diff
@@ -6,12 +6,13 @@ envlist =
 [testenv]
 deps =
 commands =
-    pip install flake8==2.1.0 pep8==1.5.6
+    pip install flake8==3.6.0 pycodestyle==2.4.0
```

Comment:
There's no need to pin pycodestyle here and the installation should happen in `deps = flake8`